js-miniapp-sdk/src/modules/secure-storage.ts

```diff
@@ -0,0 +1,50 @@
+import { MiniAppError } from '../../../js-miniapp-bridge/src';
+import { getBridge } from '../utils';
+import {
+  MiniAppSecureStorageKeyValues,
+  SecureStorageResponseStatus,
+  MiniAppSecureStorageSize,
+} from '../../../js-miniapp-bridge/src/types/secure-storage';
+
+interface SecureStorageProvider {
+  setItems(
+    items: MiniAppSecureStorageKeyValues
+  ): Promise<SecureStorageResponseStatus>;
```

Comment:
I don't think this should use `SecureStorageResponseStatus`. If the request was unsuccessful, then an error will be thrown to the `.catch`. It should maybe just return `undefined` in the promise success.

Reply:
Well, I was not feeling comfortable with returning just `undefined`. So thought of adding some type of response. I think we can add it later